tests/chia_log/handlers/test_farmer_server_handler.py

```diff
@@ -0,0 +1,44 @@
+# std
+import unittest
+from pathlib import Path
+
+# project
+from src.chia_log.handlers import farmer_server_handler
+from src.notifier import EventType, EventService, EventPriority
+
+
+class TestFarmerServerHandler(unittest.TestCase):
+    def setUp(self) -> None:
+        self.handler = farmer_server_handler.FarmerServerHandler()
+        self.example_logs_path = Path(__file__).resolve().parents[1] / "logs/farmer_server"
+
+    def testNominal(self):
+        with open(self.example_logs_path / "nominal.txt") as f:
+            logs = f.readlines()
+
+        for log in logs:
+            events = self.handler.handle(log)
+            self.assertEqual(len(events), 0, "Not expecting any events")
+
+    def testSkippedSignagePoints(self):
+        with open(self.example_logs_path / "disappearing_harvester.txt") as f:
+            logs = f.readlines()
+
+        expected_messages = [
+            "Remote harvester offline: 178.19.176.201 did not participate for 385 seconds!",
```

Comment:
I'd suggest replacing any real looking IP addresses with obviously dummy ones for tests. E.g. `255.255.255.255` or ones from local network ranges `192.168.X.X`

Reply:
Fixed in bf28871